plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -930,7 +930,7 @@ def _publish_distribution_packages_link(self, distribution_unit):
                 # a package path exists as a symlink we are going to remove it since
                 # the _create_symlink will create a real directory
                 os.unlink(package_path)
-        default_dir = self.get_config().get('packages_directory', 'Packages')
+        default_dir = self.get_config().get('packages_directory', '')
```

Comment:
We cannot do a code change for 2.9, only docs. This needs to be in a separate commit, or it needs to be removed from this PR and the docs that go into the 2.9 release need to be updated to reflect what happens without this change in place.